packages/gitbook-docs/src/parseInPageMenu.ts

```diff
@@ -0,0 +1,29 @@
+import Markdoc, { ConfigType, RenderableTreeNode } from '@markdoc/markdoc';
+import { document } from './markdoc/schema/document';
+import { heading } from './markdoc/schema/heading';
+import { parseAst } from './parseContent';
+
+export type ParseInPageMenuConfig = {
+  readonly assetsPrefix: string;
+  readonly pagePath: string;
+};
+
+const HEADING_LEVELS_TO_SHOW = [2, 3];
```

Comment:
Could you add a comment about this `const`? Why the values are just `2` and `3`?